gui/ThemeEngine.cpp

```diff
@@ -876,6 +876,10 @@ void ThemeEngine::drawDD(DrawData type, const Common::Rect &r, uint32 dynamic, b
 		extendedRect.clip(_clip);
 	}
 
+	if (extendedRect.isEmpty()) {
+		return;
+	}
```

Comment:
Under what circumstances could this occur?

Reply:
When the scroll container widget is scrolled down, some the widgets it contains are outside of the visible area.